corehq/util/django_migrations.py

```diff
@@ -110,3 +113,40 @@ def noop_migration():
     Old migrations can be simply removed. `operations = []` is perfectly valid
     """
     return RunPython(RunPython.noop, RunPython.noop)
+
+
+def run_management_command(command_name, *args, required_commit=None, **kwargs):
+    """Return a migration operation that can be used to run a management command from
+    a Django migration. This will give the user directions for running the command
+    manually if the command fails or has since been removed from the codebase"""
+    @skip_on_fresh_install
```

Comment:
I wonder if this behavior should be made more explicit, though I can't think of a great way to do that.  I'm thinking of bootstrap type commands that _should_ run on a fresh install, but which use management commands.  An overwhelming majority of usages will want this behavior, though.  Just wanted to raise, feel free to act as you see fit.

Reply:
4d90a8b16f3